ingest/core/query_test.py

```
@@ -0,0 +1,209 @@
+    print("-" * 50)
+
+    while True:
+        try:
+            query = input("\nQuery: ").strip()
+
+            if query.lower() in ["quit", "exit", "q"]:
+                break
+            elif query.lower() == "stats":
+                stats = vector_store.get_index_stats()
+                print("\n📊 Index Statistics:")
+                for key, value in stats.items():
```

Ruff (0.13.1)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)